### DISCUSSION TITLE
Add roller top conveyor

### DISCUSSION THREAD

**Discussion by tiagocunico** [Open-Industry-Project/Open-Industry-Project#78] [Ideas]
I think it would be nice to implement this types of conveyor. 

This one is important to simulate the accumulation of the system.
![image](https://github.com/user-attachments/assets/5e2fc0ae-7bea-4789-8975-296a31bd266e)

These are important working on transfers
![image](https://github.com/user-attachments/assets/88a43710-b971-4cfd-b3b5-0acfa2feae48)
![image](https://github.com/user-attachments/assets/4eee8382-572b-4f2a-8096-01d3b3fd8788)

All images i got [here](https://www.beltpower.com/wp-content/uploads/2022/08/Intralox-Engineering-Manual-2020.pdf)

**Comment by ryevdokimov** [comment]
Thanks for the suggestion. Adding more types of parts like this is definitely on the road map.

**Comment by tiagocunico** [comment]
I was tinkering about, and discovered that in the RigidBody3D of the conveyor we can change the friction of the conveyor, I test and it works perfectly. I'm not sure the best way to implement, but if I could decide the friction for each conveyor or change like a combo box setting, and it changes the friction, would be amazing.

**Reply by ryevdokimov** [reply]
You can modify the RigidBody3D in instances of parts individually by enabling right clicking on the instance in your simulation scene and enabling "Editable Children" 

![image](https://github.com/user-attachments/assets/9f4afd15-4605-4df2-8081-f49b34da7f6c)
